### PR TITLE
Add the addon manager and dashboard to the iso.

### PR DESCRIPTION
**Pull request by dlorenc**

### CODE REVIEW

**Comment by dlorenc** [comment]
Output with this running:
$ kubectl get pods --all-namespaces
NAMESPACE     NAME                           READY     STATUS    RESTARTS   AGE
kube-system   kube-addon-manager-127.0.0.1   1/1       Running   0          30s
kube-system   kubernetes-dashboard-txpah     1/1       Running   0          7s

**Comment by dlorenc** [comment]
I'll cut the release after this goes in.

File: deploy/addons/addon-manager.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
```
Comment (luxas) [inline deploy/addons/addon-manager.yaml]: This file is already present in deploy/static-pods/addon-manager, please copy that one to avoid duplication

Comment (luxas) [inline deploy/addons/addon-manager.yaml]: Ah, didn't notice the volume mounting difference between the two.
Then you may move this to `deploy/iso/addon-manager.yaml` or `deploy/iso/static-pods/addon-manager.yaml`

Also, please move `deploy/static-pods/addon-manager.yaml` to `deploy/docker/static-pods/addon-manager.yaml`

File: deploy/addons/addon-manager.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+kind: Pod
+metadata:
+  name: kube-addon-manager
+  namespace: kube-system
+  version: v1
+spec:
+  hostNetwork: true
+  containers:
+  - name: kube-addon-manager
+    image: gcr.io/google-containers/kube-addon-manager:v1
```
Comment (luxas) [inline deploy/addons/addon-manager.yaml]: `kube-addon-manager-amd64:v2`

**Comment by dlorenc** [comment]
Moved the files as requested.

File: deploy/iso/addon-manager.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+kind: Pod
+metadata:
+  name: kube-addon-manager
+  namespace: kube-system
+  version: v1
+spec:
+  hostNetwork: true
+  containers:
+  - name: kube-addon-manager
+    image: gcr.io/google-containers/kube-addon-manager:v1
```
Comment (luxas) [inline deploy/iso/addon-manager.yaml]: It should be `gcr.io/google-containers/kube-addon-manager-amd64:v2`
(I made the image with some bugfixes, e.g. allow daemonsets and a kubectl deprecation bugfix...)

**Comment by dlorenc** [comment]
Changed the image name.

**Comment by luxas** [comment]
LGTM

**Pull request merged**